src/adapter/src/catalog/builtin.rs

```diff
@@ -4349,6 +4376,15 @@ ON mz_internal.mz_cluster_replica_metrics (replica_id)",
     is_retained_metrics_object: true,
 };
 
+pub const MZ_OBJECT_LIFETIMES_IND: BuiltinIndex = BuiltinIndex {
+    name: "mz_object_lifetimes_ind",
+    schema: MZ_INTERNAL_SCHEMA,
+    sql: "CREATE INDEX mz_object_lifetimes_ind
+IN CLUSTER mz_introspection
+ON mz_internal.mz_object_lifetimes (id, object_type)",
+    is_retained_metrics_object: true,
```

Comment:
Doesn't this retain history on the table for 30 days? I don't think we want that behavior on this object.

Reply:
Good catch, this was a copy-paste error, updated!